tools/build-tool/castle-engine.dpr

```diff
@@ -179,9 +180,16 @@ begin
             'output' +NL+
             '    Output some project information (from the manifest).' + NL +
             '    Next parameter determines the information:' + NL +
+            '      executable-name' + NL +
             '      version' + NL +
             '      version-code' + NL +
             NL+
+            'output-environment' +NL+
+            '    Output some environment informations.' + NL +
```

Comment:
`Output some environment informations.`
->
`Output some environment information (independent of any project).`